src/styles/components/_modal.scss

```diff
@@ -0,0 +1,76 @@
+.bsds-modal
+{
+  box-shadow: $box-shadow-primary;
+  border: 0;
+  padding: space(4x);
+  background: $neutral-100;
+  max-width: 90vw;
+  min-width: min(600px, 100vw);
```

Comment:
How do we want to handle this value? Leave a comment? Should it be the rem equivalent?

Reply:
I'd recommend using rem and // NOTE: that it's a one-off. We can keep an eye out as we build more modal/panel type components to see if this is a dimension we want to flip to a variable.